shared_kv_storage/kv_storage/remote_lmdb/shared_kv_dbstore.py

```diff
@@ -82,9 +82,10 @@ def set(self, table, key, value):
         try:
             self._db_store.db_store_put(table, key, value)
             return True
-        except Exception as ex:
-            logger.error("Exception occurred in set - {}".
-                         format(ex))
+        except Exception:
+            # @TODO : Instead of suppressing exception here, pass it back
+            # and let the caller decide how to react to the exception.
+            logger.warn("Could not set key-value in database.")
```

Comment:
Change log level to error in case of exception

Reply:
@manju956 This may not be an error case always. The caller of the API will have the right context to decide if this is an error case.